Lib/unittest/mock.py

```diff
@@ -1085,15 +1085,15 @@ def _get_child_mock(self, /, **kw):
         return klass(**kw)
 
 
-    def _calls_repr(self, prefix="Calls"):
+    def _calls_repr(self, prefix="Calls", display_empty=False):
```

Comment:
```suggestion
    def _calls_repr(self, prefix="Calls", *,  display_empty=False):
```

Reply:
Done in https://github.com/python/cpython/pull/113573/commits/1e178389978b12aa85ff3cd1de16c6e17b45e89a